Parser/Tree_Nodes/stmt.h

```diff
@@ -28,8 +28,10 @@ typedef struct stmt
 {
     f_call * function_call;
     asmt * assignment;
+    r_asmt *re_asmt;
     expr * expression;
-}stmt;
+    b_stmt_list *bStmtList;
+} stmt;
```

Comment:
I don't think we described it in the meeting, but the stmt struct should not have a b_stmt_list, it should have elements for an if_node, for_node, and while_node, like b_stmt. The ONLY difference between the two is the lack of asmt in the b_stmt.